scripts/setup/ubuntu/install_deps.sh

```diff
@@ -14,7 +14,9 @@ DEPS=(
   g++
   gcc
   git
+  gnuplot # Litani (recommended) dependency
```

Comment:
Could you add a line to the README about running `install_deps.sh`?

Reply:
I followed @zhassan-aws advice to put this into its own script. Do you want to use `install_deps.sh` in our README like we do in `rmc-docs`?